_aggregations/pipeline/serial-diff.md

```diff
@@ -0,0 +1,292 @@
+
+Unlike sibling pipeline aggregations, which operate across all buckets and produce a single output, parent pipeline aggregations process each bucket individually and write the result back into each bucket.
+
+Use the `serial_diff` aggregation to compute changes between time periods with a specified lag. The `lag` parameter (a positive integer value) specifies which previous bucket value to subtract from the current one. The default `lag` value is `1`, meaning `serial_diff` subtracts the value in the immediately previous bucket from the value in the current bucket.
+
+## Parameters
+
+The `serial_diff` aggregation takes the following parameters:
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `buckets_path`        | Required          | String          | The path of the aggregation buckets to be aggregated. See [Pipeline aggregations]({{site.url}}{{site.baseurl}}/aggregations/pipeline/index#buckets-path). |
+| `gap_policy`          | Optional          | String          | The policy to apply to missing data. Valid values are `skip`, `insert_zeros`, and `keep_values`. Default is `skip`. |
+| `format`              | Optional          | String          | A [DecimalFormat](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/text/DecimalFormat.html) formatting string. Returns the formatted output in the aggregation's `value_as_string` property. |
+| `lag`                 | Optional          | Integer         | The historical bucket to subtract from the current bucket. Must be a positive integer. Default is `1`. |
+
+## Example
+
+The following example creates a date histogram with a one-month interval from the OpenSearch Dashboards logs sample data. The `sum` sub-aggregation calculates the sum of all bytes for each month. Finally, the `serial_diff` aggregation calculates month-to-month difference in total bytes from these sums:
+
+```json
+GET opensearch_dashboards_sample_data_logs/_search
+{
+   "size": 0,
+   "aggs": {
+      "monthly_bytes": {                  
+         "date_histogram": {
+            "field": "@timestamp",
+            "calendar_interval": "month"
+         },
+         "aggs": {
+            "total_bytes": {
+               "sum": {
+                  "field": "bytes"     
+               }
+            },
+            "monthly_bytes_change": {
+               "serial_diff": {                
+                  "buckets_path": "total_bytes",
+                  "lag": 1
+               }
+            }
+         }
+      }
+   }
+}
+```
+{% include copy-curl.html %}
+
+The response contains the month-to-month difference for the second and third month. (The first month `serial_diff` cannot be calculated because there's no previous month to compare with):
```

Comment:
```suggestion
The response contains the month-to-month difference for the second and third months. (The first month `serial_diff` cannot be calculated because there's no previous month against which to compare it):
```